api.planx.uk/admin/session/oneAppXML.ts

```diff
@@ -21,7 +21,8 @@ const fetchSessionXML = async (sessionId: string) => {
     const query = gql`
       query GetUniformApplicationBySessionID($submission_reference: String) {
         uniform_applications(
-          where: { submission_reference: { _eq: $submission_reference } }
+          where: { submission_reference: { _eq: $submission_reference } },
+          order_by: { created_at: desc }
```

Comment:
nit: `limit: 1` might also help communicate explicit intention here, even though we `return uniform_applications[0]` later?

Reply:
No this is a great comment - I agree that `created_at` is not particularly meaningful on its own. Updated!